pkg/agent/testdata/AKSUbuntu1604+K8S117/line48.sh

```diff
@@ -56,8 +56,8 @@ downloadGPUDrivers() {
     fi
 
     mkdir -p ${GPU_DEST}
-    retrycmd_if_failure 30 5 3600 apt-get install -y linux-headers-$(uname -r) gcc make dkms || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
-    retrycmd_if_failure 30 5 60 curl -fLS https://us.download.nvidia.com/tesla/$GPU_DV/NVIDIA-Linux-x86_64-${GPU_DV}.run -o ${GPU_DEST}/nvidia-drivers-${GPU_DV} || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
+    retrycmd_if_failure 30 5 3600 apt-get install -y linux-headers-$(uname -r) gcc make dkms || exit $ERR_GPU_DRIVERS_DOWNLOAD_TIMEOUT
```

Comment:
I replaced `ERR_GPU_DRIVERS_INSTALL_TIMEOUT` with the new `ERR_GPU_DRIVERS_DOWNLOAD_TIMEOUT` here, as it is in the `downloadGPUDrivers` function. 

Would `ERR_APT_INSTALL_TIMEOUT` or another error code be more appropriate?

Reply:
sgtm

Reply:
Replaced this with `ERR_APT_INSTALL_TIMEOUT`.